spec/draft-barnes-cfrg-4q.md

```diff
@@ -175,31 +172,34 @@ efficient formulas for addition and doubling have substantially
 reduced the cost of elliptic curve operations in terms of field
 operations, the number of group operations involved in scalar
 multiplication has not been reduced in the curves considered for IETF
-use. Using curves with efficiently computable endomorphisms reduces
+use. Using curves with an efficiently computable endomorphism reduces
 the number of group operations by turning one long scalar
-multiplication into the sum of two multiplications by scalars of half
+multiplication into the sum of two multiplications by scalars of half 
 the length, which can be evaluated more efficiently.
 
 By using curves over quadratic extensions there are more endomorphism
 families to pick from, and the field operations become more efficient
-compared to prime fields of the same size. With more distinct
-endomorphisms it becomes possible to divide scalars into four
-parts. The field GF((2^127-1)^2) offers extremely efficient arithmetic
-as the modulus is a Mersenne prime. Together these improvements
-substantially reduce computation time compared to other proposed
-Diffie-Hellman key exchanges, but it was not possible to combine them
-all, as the demand for two distinct endomorphisms significantly restricts
-the curves that can be used.
-
-This recently changed after careful examination of known families of
-curves with two endomorphisms.  As described in {{Curve4Q}}, Curve4Q
+compared to prime fields of the same size.  The ideal case is given by curves 
+equipped with two distinct endomorphisms, so that it becomes possible to divide 
+scalars into four parts, and defined over the field GF((2^127-1)^2), which
+offers extremely efficient arithmetic as the modulus is a Mersenne prime. 
+Together, these improvements substantially reduce computation time compared 
+to other proposed Diffie-Hellman key exchange and digital signature schemes.  
+However, the combined availability of these features severely restricts
+the curves that can be used for cryptographic applications.
+
+As described in {{Curve4Q}}, Curve4Q
 is the only known elliptic curve that permits a four dimensional
 decomposition over the highly efficient field GF(p^2) with p = 2^127 -
-1 and has a prime order subgroup of order approximately 2^246. No
-known elliptic curve with such a decomposition has a larger prime
-order subgroup over this field.  This "uniqueness" allays concerns
+1 and has a prime order subgroup of order approximately 2^246, which provides
+around 128 bits of security. No known elliptic curve with such a decomposition 
+has a larger prime order subgroup over this field.  This "uniqueness" allays concerns
 about selecting curves vulnerable to undisclosed attacks.
 
+Curve4Q also offers high flexibility to realize high-speed cryptographic schemes
+including Diffie-Hellman key exchange and digital signatures (e.g., see {{SchnorrQ}}). 
+This document focuses on Diffie-Hellman key exchange.
```

Comment:
Since we don't go into signature here, might rephrase this as "This document focuses on DH with this curve, but it is also possible to define a signature scheme".